### ISSUE TITLE
Protege failed to open ontology file after working with OntopPro

### ISSUE THREAD

**Issue by johardi** [ontop/ontop#104]
Steps to reproduce:
1. Create a new ontology
2. Create a class, e.g., A.
3. Create annotation assertion for class A, e.g., rdfs:comment "This is just a testing". Add a language tag: "en".
4. Switch to "Ontop Mappings" tab and create a datasource (can be just dummy input parameters)
5. Select "Test Connection" (and ignore the error message)
6. Save the ontology, e.g., test.owl. OntopPro will create an extra file test.obda
7. Close test.owl
8. Re-open test.owl
9. Create another class, e.g., B
10. Save the ontology
11. Close test.owl
12. Re-open test.owl
13. An exception will be thrown: "The prefix "xml" cannot be bound to any namespace other than its usual namespace; neither can the namespace for "xml" be bound to any prefix other than "xml"."

This steps might look arbitrary but the purpose is to have an .obda file (step 4, 5) and then make an ontology modification after working with the plugin (step 6-12). The focus of the error is the introduction of xml:lang in the ontology (step 3).

**Comment by johardi** [comment]
In addition, I found inconsistency between OBDA and OWL/Protege for `xml` prefix label.

OBDA: `xml=http://www.w3.org/XML/1998/namespace#`
OWL/Protege: `xml=http://www.w3.org/XML/1998/namespace`

The prior is more correct (reference: https://www.w3.org/TR/xml-names11/)
```
The prefix xml is by definition bound to the namespace name http://www.w3.org/XML/1998/namespace. 
```

**Comment by skomlaebri** [comment]
Fixed in 3a464de6e9d9e29927f0ec2ec6ea8b730e6000e1
